### DISCUSSION TITLE
Restoring Previous Behavior for 'important' Configuration Without :is() Selector in Latest Tailwind CSS Version

### DISCUSSION THREAD

**Discussion by Filip1139** [tailwindlabs/tailwindcss#12636] [Help]
Hello Tailwind CSS Community,

I've recently updated to the latest version of Tailwind CSS and encountered an issue regarding the `important` configuration option. In previous versions of Tailwind, I used the following setting in my `tailwind.config.js` to ensure that Tailwind's styles took precedence:

```javascript
important: '.tw',
```

This configuration wrapped all Tailwind-generated classes with a `.tw` parent, resulting in CSS like:

```css
.tw .h-2 { /* ... */ }
.tw .bg-red { /* ... */ }
```

This worked great for my projects, especially when targeting older browsers. However, after updating to the recent Tailwind CSS version, I noticed that the generated CSS now uses the `:is()` pseudo-class, which looks like this:

```css
.tw :is(.h-full) { /* ... */ }
```

The introduction of the `:is()` pseudo-class has caused compatibility issues with older browsers that don't support this feature (Safari 13)

Is there any recommendation on how to achieve this with the latest version of Tailwind CSS? I'd like to avoid manually post-processing the CSS if possible.

Thank you for your assistance.

**Comment by wongjn** [comment]
You could consider downgrading to an older Tailwind version, before the introduction of the `:is()` wrapper. Otherwise, you'd have to consider post-processing the CSS.
